### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by waqaskanju**
In this pull routes are added to calculator app.
Styling is also added.

Preview 1:
image

Preview 2:
image

Preview 3:
image

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        {' '}
+        |
+        {' '}
+        <Link to="/calculator">Calculator</Link>
+        {' '}
+        |
+        {' '}
+        <Link to="/quote">Quote</Link>
+      </nav>
+    </div>
```
Comment (danmainah) [inline src/components/Navigation.js]: - [ ] Kindly ensure your layout matches the design given currently it doesn't below is a comparison  for more detailed look on the differences

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @waqaskanju , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Navigation.js (hunk 1 above)
Comment (waqaskanju) [reply src/components/Navigation.js]: I think this module is not for designing. its for Learning React. Any how on your request I have changed the layout. Here is a preview.
image

**Review by EricMbouwe — APPROVED** [review]
Hi @waqaskanju, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters Ok
- PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**